SEDaily-IOS/CustomTabViewController.swift

```diff
@@ -92,10 +92,8 @@ class CustomTabViewController: UITabBarController, UITabBarControllerDelegate {
     func setupTabs() {
         let layout = UICollectionViewLayout()
         let vc1 = PodcastPageViewController()
-        let vc2 = JustForYouCollectionViewController(collectionViewLayout: layout)
-        let vc3 = TopCollectionViewController(collectionViewLayout: layout)
-//        let vc2 = GeneralCollectionViewController(collectionViewLayout: layout, type: API.Types.recommended)
-//        let vc3 = GeneralCollectionViewController(collectionViewLayout: layout, type: API.Types.top)
+        let vc2 = GeneralCollectionViewController(collectionViewLayout: layout, type: .recommended)
+        let vc3 = GeneralCollectionViewController(collectionViewLayout: layout, type: .top)
         
         let icon1 = UITabBarItem(title: L10n.tabBarTitleLatest, image: #imageLiteral(resourceName: "mic_stand"), selectedImage: #imageLiteral(resourceName: "mic_stand_selected"))
```

Comment:
Perhaps it would be better to set these within the associated view controller? That way each view controller controls their own UITabBarItem.

Reply:
I'll put this in the next push